guvnor-structure/guvnor-structure-client/src/main/java/org/guvnor/structure/client/editors/repository/clone/CloneRepositoryViewImpl.java

```diff
@@ -272,7 +272,7 @@ public void setCancelEnabled( final boolean enabled ) {
 
     @Override
     public void setPopupCloseVisible( final boolean closeVisible ) {
-        setPopupCloseVisible( closeVisible );
+        setClosable( false );
```

Comment:
What becomes of the unused parameter ```closeVisible```?

Either pass the parameter to ```setClosable()``` or remove from the method signature.